docs/features/circulation.md

```diff
@@ -0,0 +1,15 @@
+# Loan Circulation with Delivery Options and State Tracking
```

Comment:
```suggestion
# Loan Circulation with delivery options and state tracking
```

Reply:
Agree, I noticed the camel case in the titles in the other PRs too. To be lowercased everywhere